File: diemJava/src/main/java/diem/model/Block.java
package diem.model;

public class Block {

  public Author author;
  public int round;
  public Payload payload;
  public QuorumCertificate quorumCertificate;
  public int id;

  public Block(Author author, int round, Payload payload, QuorumCertificate quorumCertificate,
      int id) {
    this.author = author;
    this.round = round;
    this.payload = payload;
    this.quorumCertificate = quorumCertificate;
    this.id = id;
  }
}

File: diemBFT/test2/block.py
class Block:
    def __init__(self, round, author, payload):
        self.round = round
        self.id = '123'
        self.author = author
        self.payload = payload

    def __str__(self):
        return 'Round: {}, Payload: {}'.format(self.round, self.payload)
